src/orca/services/nextflowtower/models.py

```diff
@@ -2,14 +2,68 @@
 from collections.abc import Collection
 from dataclasses import field
 from datetime import datetime
-from typing import Any, Optional
+from typing import Any, Optional, Self
 
 from pydantic.dataclasses import dataclass
 
 
+@dataclass(kw_only=False)
+class User:
+    """Nextflow Tower user."""
+
+    id: int
+    username: str
+    email: str
+
+    @classmethod
+    def from_response(cls, response: dict[str, Any]) -> Self:
```

Comment:
Nit: https://docs.pydantic.dev/usage/models/#helper-functions.  Would this be helpful? (in particular the `parse_obj` function included in `BaseModels`

Reply:
That doesn't work for most models because the API responses use camel case and the data classes use snake case. 

We could make this work if we used camel case for our data classes. Not sure if we want to break convention though.

Reply:
Ya... let's not break the convention. Was just curious if we could reuse something.